tests/scripts/quiet/quiet

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
```

Comment:
Why is this executable, rather than a shell library sourced by the others?

So: no shebang in `quiet`, it's not executable, and each caller is
```
#!/usr/bin/env bash
NO_SILENCE=…
TOOL=…
"$(dirname "$0")/quiet.bash" "$@"
```